src/components/MapProps.tsx

```diff
@@ -18,6 +18,8 @@ export default interface MapProps {
 
   /**
    * The map’s color scheme when displaying standard or muted standard map types.
+   * Offers the Options ColorScheme.Dark, ColorScheme.Light and ColorScheme.Auto
+   * Auto Color Theme does decide on the colorscheme for mapkit based on the browser preference
```

Comment:
A small suggestion about grammar/wording:

```suggestion
   * Offers the options ColorScheme.Dark, ColorScheme.Light and ColorScheme.Auto.
   * Auto will select automatically the appropriate MapKit color scheme based on the browser preference.
```